rootfs-builder/euleros/Dockerfile.in

```diff
@@ -12,9 +12,32 @@ yum \
 git \
 make \
 gcc \
-coreutils \
+gcc-c++ \
+autoconf \
+automake \
+m4 \
+wget \
+tar curl \
+glibc-devel glibc-static glibc-utils glibc-headers glibc-common \
+libstdc++-devel libstdc++-static \
+coreutils which binutils vim \
+coreutils sed \
 chrony \
 curl
 
 # This will install the proper golang to build Kata components
+# cmake is too old, install the latest cmake
```

Comment:
I don't see an `install cmake` line - am I missing it ?

Reply:
No, we cannot build rust agent on euleros. Since we cannot build musl on euleros, install cmake is meanless. However, the comment is misleading..